packages/clerk-js/src/ui.retheme/components/OrganizationList/UserMembershipList.tsx

```diff
@@ -35,8 +36,12 @@ export const MembershipPreview = withCardStateProvider((props: { organization: O
 export const PersonalAccountPreview = withCardStateProvider(() => {
   const card = useCardState();
   const { hidePersonal, navigateAfterSelectPersonal } = useOrganizationListContext();
-  const { isLoaded, setActive } = useCoreOrganizationList();
-  const user = useCoreUser();
+  const { isLoaded, setActive } = useOrganizationList();
+  const { user } = useUser();
+
+  if (!user) {
+    return null;
+  }
```

Comment:
❓ is this a fix to avoid rendering `PersonalAccountPreview` when user is not available or the `useCoreUser()`  has different behaviour from the `useUser()` ?

Reply:
OrganizationList is wrapped in  `withCoreUserGuard` so we are safe. It doesn't really matter if we do `user?.something` or return null.

Here we did that because it was helping with the deconstruction.